composer.json

```diff
@@ -15,19 +15,20 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.9",
-        "drupal/admin_toolbar": "^3.0",
-        "drupal/config_split": "^1.7",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-recommended": "^9",
-        "drupal/devel": ">=4.1",
-        "drupal/workbench": "^1.3",
-        "drupal/workbench_tabs": "^1.5"
+        "composer/installers": "^2.0",
+        "drupal/admin_toolbar": "^3",
+        "drupal/config_split": "^1",
```

Comment:
I had to use config_split ^2 in another Drupal 10 project I set up. Not sure if we'll need to do that here as well.